sdks/java/extensions/avro/build.gradle

```diff
@@ -65,6 +65,7 @@ dependencies {
   }
   implementation library.java.error_prone_annotations
   implementation library.java.avro
+  implementation "org.tukaani:xz:1.9" // marked as optional in avro
```

Comment:
previously it was a test dependency, what caused this get needed in name scope now ?

Reply:
oh, I think I missed that the `avroVersion` configuration is a test configuration, I thought that starting with a certain avro version we also start depending on that library, I'll move it to a testImplementation 

I guess that would also explain why I got the unusedDeclaredArtifacts error in one of the checks

Reply:
This is for avro compatibility test suite, each test target uses a "avroVersion$k" configuration to manage its own dependencies